script/QA/QA_11.txt

```diff
@@ -467,7 +467,8 @@ pieces
 [sha:817bde2f8e9224242926f3bb3f0b9e9eb87fa6b1]{
 -- Page 284, Offset 43280.
 -- 切り刻まれての<出血死|ブレイド>がお好みですか？
- and <bleed| b l a d e  > to death just like the
+// halp spacing
+ and <die via hemorrhage| b l a d e  > just like the
```

Comment:
kinda like having the death outside the ruby because it makes the 'blade' section at least _plausibly_ connect to the sentence. 'Blade to death' is stupid but at least it _kinda_ conveys something. I.e,
> and <hemorrhage| b l a d e > to death just like the

Reply:
Sure
Honestly, as long as the reference is clear (this is before Blade is introduced IIRC), it's all good